src/functions/accentunder.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+// Horizontal overlap functions
+import defineFunction from "../defineFunction";
+import buildCommon from "../buildCommon";
+import mathMLTree from "../mathMLTree";
+import stretchy from "../stretchy";
+
+import * as html from "../buildHTML";
+import * as mml from "../buildMathML";
+
+defineFunction({
+    type: "accentUnder",
+    names: [
+        "\\underleftarrow", "\\underrightarrow", "\\underleftrightarrow",
+        "\\undergroup", "\\underlinesegment", "\\utilde",
+    ],
+    props: {
+        numArgs: 1,
+    },
+    handler: (context, args) => {
+        const base = args[0];
+        return {
+            type: "accentUnder",
+            label: context.funcName,
+            base: base,
+        };
+    },
+    htmlBuilder: (group, options) => {
+        // Treat under accents much like underlines.
+        const innerGroup = html.buildGroup(group.value.base, options);
+
+        const accentBody = stretchy.svgSpan(group, options);
+        const kern = (/tilde/.test(group.value.label) ? 0.12 : 0);
```

Comment:
nit: are there functions other than \\utilde that would match this and we just haven't implemented them yet? If not, this seems like a silly place to use a RegExp. Maybe change to:

```
const kern = (group.value.label === '\\utilde') ? 0.12 : 0;
```

Reply:
Good catch.